index.js

```diff
@@ -279,6 +279,24 @@ function runDataIntegrityProofFormatTests({
           }
         }
       });
+
+    describe('2.6 Resource Integrity', function() {
+      describe('with optional associated object named "image"', function() {
+        describe('when "digestMultibase" exists', () => {
+          it('MUST be a string or array of strings', () => {
+            this.cell = {columnId, rowId: this.title};
+            if(data.image?.digestMultibase) {
```

Comment:
if `image` required on a VC? should that be `data?.image?.digestMultibase`

Reply:
is `data` optional?